app/src/main/java/com/github/polybooks/utils/BookUtil.kt

```diff
@@ -1,6 +1,38 @@
 package com.github.polybooks.utils
 
 import com.github.polybooks.core.Book
+import com.github.polybooks.core.Course
+import com.github.polybooks.core.Field
+import com.github.polybooks.core.Semester
 
 fun anonymousBook(title: String): Book =
-        Book("", null, title, null, null, null, null, null)
\ No newline at end of file
+        Book("", null, title, null, null, null, null, null)
+
+val mockFields : List<Field> = listOf(
```

Comment:
Il y aurait moyen de mettre ça dans la DummyInterestDatabase

Reply:
Courld we merge it as it is and fix it when we will do the build-variant thing?

Reply:
okay its done